src/components/Message/Message.tsx

```diff
@@ -8,7 +8,7 @@ export interface Props {
   type: 'page' | 'toast';
   priority: 'info' | 'critical' | 'warning' | 'success';
   title?: string | string[];
-  body: string | string[];
+  body: string | (string | JSX.Element)[];
```

Comment:
can we make this 
`string | JSX.Element | (string | JSX.Element)[];`

thanks for doing this

Reply:
Good idea!